lit/v5.2.0.lit

```diff
@@ -106,6 +110,28 @@
   the dashboard page by setting \code{cluster-name} flag.
 }
 
+\note{feature}{
+  \ghuser{cappyzawa} added a new `get-team` subcommand to `fly`. It allows you to
+  retrieve a single team's config.
+}
+
+\note{fix}{
+  We reduced the default concurrency settings for volume sweeping from 5 to 3
+  as a way of reducing the stress that volume deletion end up putting on the
+  system in some cases.
+}
+
+\note{fix}{
+  \ghuser{edtan} submitted a fix when you run \code{concourse web} without a
+  \code{--session-signing-key}, it will now generate key pair to use instead.
```

Comment:
It actually already did this, but the code checking if it was `nil` choked on an intermediate `nil` field resulting in a panic. Kind of awkward to word that though.